docs/administration/managing-infrastructure/performance/index.md

```diff
@@ -72,50 +92,69 @@ Instead, we decided to put this control into your hands, allowing you to control
-    - Listening Tentacles and SSH place the Octopus Server under less load.
-    - We try to make Polling Tentacles as efficient as possible. However, they can place the Octopus Server under high load, just handling incoming connections.
-1. Reduce the frequency and complexity of automated health checks using [machine policies](/docs/infrastructure/deployment-targets/machine-policies.md).
-1. Disable automatic indexing of the [built-in package repository](/docs/packaging-applications/package-repositories/index.md) if not required.
+You can scale out your Octopus Server by implementing a [High Availability](/docs/administration/high-availability/index.md) cluster. Each node in the cluster will have its own task cap.  Two servers in a cluster each with a task cap of 5 means you can process 10 concurrent tasks.  
+
+In addition to linearly increasing the performance of your cluster, you can perform certain kinds of maintenance on your Octopus Servers without incurring downtime.
+
+### Workers
+
+Consider using [Workers](/docs/infrastructure/workers/index.md) and worker pools if deployment load is affecting your server.  See this [blog post](https://octopus.com/blog/workers-performance) for a way to begin looking at workers for performance.
+
+### Spaces
+
+Consider separating your teams/projects into "spaces" using the [Spaces](/docs/administration/spaces/index.md) feature.  A space is a "hard wall," each space has its own environments, projects, deployment targets, packages, machine policies, etc.  The only thing shared is users and teams.  That means less data for the Octopus UI to query.  Splitting 60 projects evenly into 3 spaces will result in the dashboard only having to load 20 projects instead of 60.
+
+## Tentacles
+
+Prefer [Listening Tentacles](/docs/infrastructure/deployment-targets/windows-targets/tentacle-communication.md#listening-tentacles-recommended) or [SSH](/docs/infrastructure/deployment-targets/linux/ssh-target.md) instead of [Polling Tentacles](/docs/infrastructure/deployment-targets/windows-targets/tentacle-communication.md#polling-tentacles) wherever possible.  Listening Tentacles and SSH place the Octopus Server under less load.  We try to make Polling Tentacles as efficient as possible. However, they can place the Octopus Server under high load, just handling incoming connections.
+
+Reduce the frequency and complexity of automated health checks using [machine policies](/docs/infrastructure/deployment-targets/machine-policies.md).
+
+## Packages {#package-transfer}
+
+Transfering packages from your Octopus Deploy server is a key piece of functionality in Octopus Deploy.  Transferring packages can also have a impact on your Octopus Server's performance.
```

Comment:
```suggestion
Transferring packages from your Octopus Server is a key piece of functionality when executing deployments and runbooks.  This can also have an impact on your Octopus Server's performance.
```